Documentation/ceph-object-multisite.md

```diff
@@ -124,3 +241,36 @@ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-g
 radosgw-admin zone rm --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
 radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
 ```
+
+When a zone is deleted, the pools for that zone are not deleted.
+
+### Deleting Pools for a Zone
+
+The Rook toolbox can delete pools. Deleting pools should be done with caution.
+
+If you do intend to deletepools you should read the following [documentation](https://docs.ceph.com/docs/master/rados/operations/pools/) on pools first.
+
+When a zone is created the following pools are created for each zone:
+```console
+$ZONE_NAME.rgw.control
+$ZONE_NAME.rgw.meta
+$ZONE_NAME.rgw.log
+$ZONE_NAME.rgw.buckets.index
+$ZONE_NAME.rgw.buckets.non-ec
+$ZONE_NAME.rgw.buckets.data
+```
+Here is an example command to delete the .rgw.buckets.data pool for zone-a.
+
+```console
+ceph osd pool rm zone-a.rgw.buckets.data zone-a.rgw.buckets.data --yes-i-really-really-mean-it
+```
+
+### Removing an Object Store from a Zone
+
+When an object-store (created in a zone) is deleted, the endpoint for that object store is removed from that zone, via
+```console
+kubectl delete -f object-store.yaml
+```
+
+Removing object store(s) from the master zone of the master zone group should be done with caution. When all of theses object-stores are deleted the period cannot be updated and that realm cannot be pulled.
```

Comment:
@travisn Am I clear about the ramifications of removing the object-stores from the master-zones?